man/ocitools-generate.1.md

```diff
@@ -137,18 +137,19 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
+**--poststart**=[CMD[:ARG]]
```

Comment:
On Tue, Aug 16, 2016 at 08:41:17PM -0700, Ma Shimiao wrote:
> > **--bind**=*[[HOST-DIR:CONTAINER-DIR][:OPTIONS]]*
> bind also used brackets around all. Based on your opinion, it's also need to fix? 

Yeah, I'd rather see:

  **--bind**=SOURCE:DESTINATION[:OPTIONS]

with text in the description saying something like:

  `--bind` may be set multiple times to configure multiple bind mounts.